### PR TITLE
add a shortcut to user's profile page to admin user details

### PR DESCRIPTION
**Pull request by techknowlogick**
For admin's who need quick access to user's details page

CleanShot 2023-10-06 at 22 13 49@2x

### CODE REVIEW

**Review by denyskon — APPROVED** [review]

**Comment by silverwind** [comment]
Why red? Red should only be dangerous actions like deletion etc.

File: templates/shared/user/profile_big_avatar.tmpl [hunk 1]
```diff
@@ -118,6 +118,13 @@
 				{{end}}
 			</li>
 			{{end}}
+			{{if and .IsSigned .IsAdmin}}
```
Comment (wxiaoguang) [inline templates/shared/user/profile_big_avatar.tmpl]: `IsAdmin` already means `IsSigned`

File: templates/shared/user/profile_big_avatar.tmpl [hunk 2]
```diff
@@ -118,6 +118,13 @@
...
+			<li class="follow">
```
Comment (wxiaoguang) [inline templates/shared/user/profile_big_avatar.tmpl]: Why `follow` ....

File: templates/shared/user/profile_big_avatar.tmpl [hunk 3]
```diff
@@ -118,6 +118,13 @@
...
+				<a class="ui basic red button" href="{{AppSubUrl}}/admin/users/{{.ContextUser.ID}}">
```
Comment (wxiaoguang) [inline templates/shared/user/profile_big_avatar.tmpl]: Why "red" (+1)

**Review by denyskon — CHANGES_REQUESTED** [review]
Please use a different colour

File: templates/shared/user/profile_big_avatar.tmpl (hunk 3 above)
Comment (techknowlogick) [reply templates/shared/user/profile_big_avatar.tmpl]: thanks. updated to green

File: templates/shared/user/profile_big_avatar.tmpl (hunk 2 above)
Comment (techknowlogick) [reply templates/shared/user/profile_big_avatar.tmpl]: copy/paste issue. fixed now

File: templates/shared/user/profile_big_avatar.tmpl (hunk 1 above)
Comment (techknowlogick) [reply templates/shared/user/profile_big_avatar.tmpl]: simplified

**Comment by techknowlogick** [comment]
updated to an icon next to username instead of a button. screenshot has been updated too

**Review by denyskon — APPROVED** [review]
A bit weird line indent in the code 😆 but I like the idea

File: templates/shared/user/profile_big_avatar.tmpl [hunk 4]
```diff
@@ -13,7 +13,11 @@
 	</div>
 	<div class="content gt-word-break profile-avatar-name">
 		{{if .ContextUser.FullName}}<span class="header text center">{{.ContextUser.FullName}}</span>{{end}}
-		<span class="username text center">{{.ContextUser.Name}}</span>
+		<span class="username text center">{{.ContextUser.Name}} {{if .IsAdmin}}
+					<a class="muted" href="{{AppSubUrl}}/admin/users/{{.ContextUser.ID}}" data-tooltip-content="{{ctx.Locale.Tr "admin.users.details"}}">
+						{{svg "octicon-gear" 18 "gt-mr-2"}}
```
Comment (wxiaoguang) [inline templates/shared/user/profile_big_avatar.tmpl]: Why `"gt-mr-2"`?

Comment (techknowlogick) [reply templates/shared/user/profile_big_avatar.tmpl]:
```suggestion
						{{svg "octicon-gear" 18}}
```

**Review by wxiaoguang — APPROVED** [review]
The indent doesn't look good to me, while it doesn't bother me.

**Pull request merged**